air/src/execution_step/value_types/stream/recursive_stream.rs

```diff
@@ -194,7 +202,7 @@ mod test {
         let mut recursive_stream = RecursiveStreamCursor::new();
 
         let value = create_value(json!("1"));
-        stream.add_value(value, Generation::Current(0.into()));
+        add_value_result_checker(stream.add_value(value, Generation::current(0)));
```

Comment:
`assert!(stream.add_value(...).is_ok())` is more idiomatic.
You may also return `Result<(), AError>` from a test, and the test will fail is `Err` is returned.

Reply:
Just calling `.unwrap()` is totally fine too.